src/Util/lang/en_US.json

```diff
@@ -12,6 +12,12 @@
     "banner": "Command Line Interface for the Nexcess.net / Thermo.io API Client\nⓒ 2018 Nexcess.net, LLC\n",
     "cloud_account": {
       "backup" : {
+        "download": {
+          "done": "Done",
```

Comment:
suggest including the local path+filename we downloaded to.
I'd actually say it's more useful here than as part of the "downloading" message, but up to you.

Reply:
added download_path to the message but not $download_path+$filename because we've not normalized it yet.